api.md

```diff
@@ -170,6 +172,70 @@ SELECT add_dimension('conditions', 'device_id', number_partitions => 2);
 SELECT add_dimension('conditions', 'device_id', number_partitions => 2, if_not_exists => true);
 ```
 
+---
+## attach_data_node() [](attach_data_node)
+
+Attach an data node to a hypertable. The data node should have been
```

Comment:
sorry - this is such a small nit - but "Attach a data node"?

Reply:
Fixed.